### DISCUSSION TITLE
a post with giscus comments | You R. Name

### DISCUSSION THREAD

**Discussion by giscus** [alshedivat/al-folio#1740] [Comments]
# a post with giscus comments | You R. Name

an example of a blog post with giscus comments

https://alshedivat.github.io/al-folio/blog/2022/giscus-comments/

**Comment by alshedivat** [comment]
testing giscus comments :sparkles:

**Reply by EmileRouxSMB** [reply]
it works

**Reply by EmileRouxSMB** [reply]
yes

**Reply by Amarnath-someperson** [reply]
damn

**Comment by arpitkaushal** [comment]
what ???? I can comment and there is no database on this app!?

**Reply by arpitkaushal** [reply]
People are creating cool stuff with jekyll!

**Reply by carloshkayser** [reply]
It uses https://giscus.app. It is powered by GitHub Discussions https://docs.github.com/en/discussions

**Comment by Sckarge** [comment]
Testing giscus comments on this post

**Comment by ammahmoudi** [comment]
nice

**Comment by winnieyangwannan** [comment]
testing :)

**Comment by elijahparker000** [comment]
test :)

**Reply by elijahparker000** [reply]
Why wasn't I already putting this on my website?

**Comment by zhyantao** [comment]
test

**Comment by orm011** [comment]
does this still work... I've been having issues on my own deployment.

**Comment by armixz** [comment]
Heey :]

**Comment by MarcDymetman** [comment]
great!

**Comment by KuchikiNamthip** [comment]
great!

**Comment by ckodser** [comment]
Cool

**Comment by iamtalhaasghar** [comment]
it works!

**Comment by alexiglad** [comment]
test
